src/v0/destinations/af/README.md

```diff
@@ -0,0 +1,332 @@
+# AppsFlyer Destination
+
+Implementation in **Javascript**
+
+## Configuration
+
+### Required Settings
+
+- **Authorization Type**: Choose between v1 (Dev Key) or v2 (Server-to-Server Key) authentication
+  - **v1**: Uses AppsFlyer Dev Key for authentication
+  - **v2**: Uses AppsFlyer Server-to-Server API Key for authentication (recommended for server-side integrations)
+
+- **AppsFlyer Dev Key** (Required for v1 authorization): Your AppsFlyer application's dev key
+  - Must be 1-100 characters long
```

Comment:
```suggestion
```
This is probably coming from the basic validations we have in config on 1-100 or 1-200 length, should this be put in doc? We can probably put this is a required field